guides/common/modules/proc_installing-the-dns-infoblox-module.adoc

```diff
@@ -15,11 +13,14 @@ DNS records are managed only in the default DNS view, it's not possible to speci
 --foreman-proxy-dns true \
 --foreman-proxy-dns-provider infoblox \
 --foreman-proxy-plugin-dns-infoblox-dns-server _infoblox.example.com_ \
---foreman-proxy-plugin-dns-infoblox-dns-view default \
+--foreman-proxy-plugin-dns-infoblox-dns-view _default_ \
 --foreman-proxy-plugin-dns-infoblox-password infoblox \
 --foreman-proxy-plugin-dns-infoblox-username admin
```

Comment:
Similar here, these are specified by the user so:
```suggestion
--foreman-proxy-plugin-dns-infoblox-password _infoblox_ \
--foreman-proxy-plugin-dns-infoblox-username _admin_
```

I'd also consider moving username/password together with server since in my mind those 3 are strongly related and values that you must always specify (unlike the view).